wp-content/themes/haruoshi_origin/functions.php

```diff
@@ -41,6 +41,28 @@ function enqueue_style() {
 			filemtime( get_stylesheet_directory() . '/dist/css/home.css' ),
 		);
 	endif;
+
+	if ( is_singular( 'news' ) ) :
```

Comment:
`is_post_type_archive()` でnewsの一覧ページのとき読み込むようにしてください。
おそらく newsの一覧ページとトップページのスタイルがほぼ一緒になると思うので、
```suggestion
	if ( is_post_type_archive( 'news' ) || isis_home() ) :
```
みたいな感じになるかと思います。

Reply:
`is_singular( 'news' )`　これはnewsの個別投稿ページのやつなんだけど、別の書き方ある？

Reply:
別の書き方はないです。

archive-xxx.php で生成されるページを一覧ページ、single-xxx.phpで生成されるページを詳細ページと呼んでいます。
デモページにはnews詳細ページがありましたが、今回は作るサイトではnews詳細ページ不要でした。
なので、news一覧ページとトップページのnewsのセクション用のスタイルをnews.scssに記述し、その上で、functions.phpの記述は
```
if ( is_post_type_archive( 'news' ) || is_home() ) :
```
となるべきでした。
news詳細ページを残したいのであれば、news詳細ページでもnews.scssの記述を適用しないといけなくなるので、
```
if( get_post_type() === 'news' || is_home() ) :
```
とかになると思います。

Reply:
https://github.com/oooshima/origin_wordpress/pull/25/commits/1dc0569ee7ac557e432bd3aa88b4e8ac309f07e5
追加してみたけど`http://localhost/origin/news/`が404になってしまうのなぜですか？

Reply:
news詳細ページは残したいので、
```
	if ( is_home() || is_post_type_archive( 'news' ) ) :
		wp_enqueue_style(
			'home',
			get_stylesheet_directory_uri() . '/dist/css/home.css',
			array(),
			filemtime( get_stylesheet_directory() . '/dist/css/home.css' ),
		);
	endif;

	if ( get_post_type() === 'news' ) :
		wp_enqueue_style(
			'news',
			get_stylesheet_directory_uri() . '/dist/css/news.css',
			array(),
			filemtime( get_stylesheet_directory() . '/dist/css/news.css' ),
		);
	endif;
```
これでいいですか！？

Reply:
ああ！理解した

Reply:
news詳細ページのcssは何て名前にすればいいの

Reply:
やっぱよくわかんなかったけどこれでいいのか？
https://github.com/oooshima/origin_wordpress/pull/25/commits/6ea2c8323d6fb0a62ae35e93791572c9ee9a5db2

Reply:
そしてこのアーカイブページのURLなに？
http://localhost/origin/news/
これだと404になるのだが

Reply:
え、てかこの構造だとやっぱりhomeにcontentってクラス必要では？
`margin-bottom: 130px;`とかの設定、アーカイブページにはいらなくてhomeでだけ必要な設定書くところがない
https://github.com/oooshima/origin_wordpress/pull/25#discussion_r820371322